docs/HowToDevelopThingsUsingSTDK.md

```diff
@@ -0,0 +1,256 @@
+# How to develop things using STDK based TizenRT   
+
+STDK is one of Develop Kit(STDK) for Smart Things Eco-system.   
+This guide shows an example how to enable STDK and develop things.   
+
+So now, it will explain how to develop Led Things of SmartThings and Control it through SmartThings Mobile app.   
```

Comment:
"So now, it will explain" -- replace with "We choose a common example of a blinking Led to demonstrate how to develop a so-called Led Things and control it via a SmartThings mobile app."

Reply:
Has Changed.